### PR TITLE
Offline user signup flow (new)

### PR DESCRIPTION
**Pull request by nickkuh**
https://trello.com/c/0eYuJU4f/121-offline-user-signup-flow-new

Screenshot 2020-04-29 at 16 24 18

Screenshot 2020-04-29 at 16 28 12

### CODE REVIEW

File: src/components/home-header.tsx [hunk 1]
```diff
@@ -33,6 +36,27 @@ export const HomeHeaderTitle = () => {
...
+          {flexDirection: 'row'},
+        ]}>
+        <Image source={iconHomeHeader} />
+        <Text
+          style={{
+            ...navigation.homeHeaderTitleStyle,
+            marginHorizontal: 11,
+          }}
+          numberOfLines={1}>
+          BP Passport
```
Comment (SamRichards) [inline src/components/home-header.tsx:54]: Should this be translated?

File: src/screens/LoginScreen.tsx [hunk 2]
```diff
@@ -20,70 +26,110 @@ type Props = {
...
+                    <FormattedMessage id="general.and" />{' '}
+                  </BodyText>
+                  <BodyText
+                    style={{color: colors.blue2}}
+                    onPress={() => {
+                      Linking.openURL('https://simple.org/digitalprinciples/')
+                    }}>
+                    <FormattedMessage id="login.terms-of-use-link" />
+                  </BodyText>
+                  <BodyText>.</BodyText>
```
Comment (SamRichards) [inline src/screens/LoginScreen.tsx:91]: I don't think this needs to be inside another BodyText tag

File: src/components/home-header.tsx (hunk 1 above)
Comment (nickkuh) [reply src/components/home-header.tsx:54]: Good spot

File: src/screens/LoginScreen.tsx (hunk 2 above)
Comment (nickkuh) [reply src/screens/LoginScreen.tsx:91]: This is from James current work I believe.

**Pull request merged**